plugins.md

```diff
@@ -42,6 +42,7 @@ Name | Description | Stars
 [oidc-login](https://github.com/int128/kubelogin) | Log in to the OpenID Connect provider | ![GitHub stars](https://img.shields.io/github/stars/int128/kubelogin.svg?label=stars&logo=github)
 [open-svc](https://github.com/superbrothers/kubectl-open-svc-plugin) | Open the Kubernetes URL(s) for the specified service in your browser. | ![GitHub stars](https://img.shields.io/github/stars/superbrothers/kubectl-open-svc-plugin.svg?label=stars&logo=github)
 [outdated](https://github.com/replicatedhq/outdated) | Finds outdated container images running in a cluster | ![GitHub stars](https://img.shields.io/github/stars/replicatedhq/outdated.svg?label=stars&logo=github)
+[passman](https://github.com/chrisns/kubectl-passman) | Provides glue between common password managers and kubectl for user credentials | ![GitHub stars](https://img.shields.io/github/stars/chrisns/kubectl-passman.svg?label=stars&logo=github)
```

Comment:
The description here is different from your manifest `shortDescription`. Although this is not a problem, you should know that this page is auto-generated and uses the `shortDescription` for the second column. So this variant is only temporary. (And your short description is good, because it gets truncated at ~50 characters)

Reply:
This is going to be in active development for a bit, wanted it on krew to get some exposure and feedback. will fix the table on my next PR, thanks!